spec.go

```diff
@@ -3,8 +3,10 @@ package apprclient
 const (
 	httpClientTimeout = 5
 
-	// status returned on a successful request to a CNR server.
+	// status returned on a successful GET request to a CNR server.
```

Comment:
okStatus

Reply:
Good catch, I've changed both to include the name of the constants.